testing/tests/DevExpress.exporter/jspdfParts/v3/jspdf_v3.dataGrid.paddings.tests.js

```diff
@@ -0,0 +1,1443 @@
+import { exportDataGrid } from 'exporter/jspdf/v3/export_data_grid_3';
```

Comment:
'padding' seems to be an additional config to existing configs.
I would prefer to review tests as 'additional' test near the source test in our common, grouping, summary, bands, and other tests.